docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -13,15 +13,15 @@ This article guides public officers to do the following:
 >1. Depending on the allotted schedule, the agency admin or cloud admin receives an email with instructions on how to register for TechPass and onboard non-GSIB and a non-DWP device to SEED.
 >1. Agency admin or cloud admin will share this registration link with the required GCC 1.0 users.
 >1. While registering, ensure to provide the correct VPN ID.
->1. It may take up to two weeks for us to verify the VPN ID before sending the TechPass invite email. If you do not receive this email after two weeks, check if it is the same email address you provided while registering for TechPass or check your Spam, Junk Email, Deleted Items or Archive folder.
+>1. It might take up to two weeks for us to verify the VPN ID before sending the TechPass invite email. If you do not receive this email after two weeks, check if it is the same email address you provided while registering for TechPass or check your Spam, Junk Email, Deleted Items or Archive folder.
```

Comment:
hmmm.. although might is not past tense. but may is best used for present tense or if there's a possibility of it happening:
https://www.masterclass.com/articles/may-vs-might-explained#:~:text=Although%20%E2%80%9Cmight%E2%80%9D%20is%20not%20the,be%20the%20better%20word%20choice.